test/DynamoCoreTests/AnalyticsTests.cs

```diff
@@ -79,55 +75,64 @@ public override void Cleanup()
         protected virtual void VerifyEventTracking(Times times)
         {
             Analytics.TrackEvent(Actions.New, Categories.NodeOperations, "New Node", 5);
+            Thread.Sleep(100);
```

Comment:
Would it be more robust to use Tasks and ContinueWith?

Reply:
Maybe. But the tasks are not returned from these Analytics methods. So I am stuck messing around with Sleep